vktrace/vktrace_layer/vktrace_lib_trace.cpp

```diff
@@ -4693,6 +4511,17 @@ VK_LAYER_EXPORT VKAPI_ATTR VkResult VKAPI_CALL vkEnumerateInstanceExtensionPrope
     return VK_ERROR_LAYER_NOT_PRESENT;
 }
 
+VKTRACER_EXPORT VKAPI_ATTR VkResult VKAPI_CALL __HOOKED_vkEnumerateInstanceExtensionProperties(const char* pLayerName,
```

Comment:
I added these to maintain consistency between what the script says is "manually hooked" and reality.